### PR TITLE
Improve logging support on Android

### PR DESCRIPTION
**Pull request by at-ninja**
Add support to use the Android NDK logging library.

### CODE REVIEW

File: src/ppx/log.cpp [hunk 1]
```diff
@@ -130,7 +134,7 @@ bool Log::CreateObjects(uint32_t modes, const char* filePath)
     //// Allocate mutex dynamically since it doesn't like being
     //// initialized before main...and lock will crash.
     ////
-    //mWriteMutex = std::make_unique<std::mutex>();
```
Comment (Keenuts) [inline src/ppx/log.cpp:133]: Not part of the CL, but since clang-format touches it, might be a good occasion to remove this comment and dead-code 😊

**Review by Keenuts — COMMENTED** [review]
Thanks for the PR! More of an high-level suggestion on my end 😊

**Comment by at-ninja** [comment]
I messed up the git history, let me re-send a new PR for this.

**Comment by vettoreldaniele** [comment]
Feel free to force push on this PR instead, if you prefer.

**Comment by at-ninja** [comment]
I was unable to reopen this PR after force-pushing, so I opened a new one with revisions from your comments.

**Pull request closed without merging**